framework/framework_test.go

```diff
@@ -221,14 +239,22 @@ func Test_Operator_ProcessDelete_ResourceOrder_Retry(t *testing.T) {
 			ErrorCount:  tc.ErrorCount,
 			ErrorMethod: tc.ErrorMethod,
 		}
-		rs := []Resource{
+		rs := []spec.Resource{
 			tr,
 		}
 		bf := func() backoff.BackOff {
 			return &backoff.ZeroBackOff{}
 		}
 
-		err := testMustNewFramework(t).ProcessDeleteWithBackoff(nil, rs, bf)
+		config := retry.DefaultWrapConfig()
+		config.BackOffFactory = bf
+		config.Resources = rs
+		wrapped, err := retry.Wrap(config)
+		if err != nil {
+			t.Fatal("test", i+1, "expected", nil, "got", err)
+		}
```

Comment:
This is how a user of the operator framework would wrap resources with the retry resource. The user is free to do it or not to do it.